components/cardsContainer/index.js

```diff
@@ -65,30 +66,33 @@ const CardsContainer = ({
     }
   }
 
-  function WelcomeCard({ openCreateShotModal }) {
+  function createProjectHandler() {
+    sendEvent(EVENTS.OPEN_CREATE_NEW_PROJECT_MODAL);
+    openCreateProjectModal();
+  }
+
+  function WelcomeCard({ openCreateShotModal, createProjectHandler }) {
```

Comment:
Please delete this component from here, in one JS file have to one component only, for other developers can read your code correct